docs/cow-protocol/concepts/batch-auctions/how-batch-auctions-work.md

```diff
@@ -0,0 +1,16 @@
+---
+sidebar_position: 5
+---
+
+# How batch auctions work
+
+Given that Ethereum settles orders in discrete blocks every 12 seconds, the CEX-native method of orderbook trading may not be the optimal mechanism for settling DeFi trades. Due to the nature of AMMs, which settle trades continuously, the order in which a transaction executes determines how good of a deal a trader is getting. Batch auctions are a DeFi-native solution that provides fair pricing for all transactions within an Ethereum block. 
+
+Ethereum has been friendly to batching mechanisms for a while. With the introduction of [PBS](https://ethereum.org/nl/roadmap/pbs/), Ethereum broke from a single-auction model into a two-sided marketplace. Builders take part in an initial auction to determine what transactions will be included in a block, and proposers take part in a second auction to decide which block will ultimately be committed to the chain next. So users are already, in fact, participating in “batch auction”-style transaction settlement. CoW Protocol simply pulls this mechanism forward to the order creation stage.
```

Comment:
PBS maximises payout for the validator (extract value from users), CoW protocol focusses on giving it back, EBBO not actually enforced pre-settlement